jupyter_client/utils.py

```diff
@@ -15,7 +19,6 @@ def wrapped(*args, **kwargs):
         except RuntimeError:
             loop = asyncio.new_event_loop()
             asyncio.set_event_loop(loop)
-        import nest_asyncio  # type: ignore
 
         nest_asyncio.apply(loop)
```

Comment:
Not needed anymore?

Reply:
I'm guessing this needs to be moved into the exception handling so the new loop is patched (assuming the global `apply` is retained).

Reply:
Yeah it's still needed, but it could have probably gone inside the exception handling